tcf_website/templates/department/department.html

```diff
@@ -18,14 +18,13 @@
         {% for sd in department.subdepartment_set.all %}
         <div class="subdepartment container-lg {% if not sd.has_current_course %}old{% endif %}">
             <h1 class="my-3">{{ sd.name }}</h1>
-            <ul class="course-list">
-                {% for c in sd.recent_courses|dictsort:"number" %}
-                    <li class="{% if not c.is_recent %}old{% endif %}">
-                        {% url 'course' course_id=c.pk as link %}
-                        {% include "common/rating_card.html" with title=c.code subtitle=c.title link=link rating=c.average_rating difficulty=c.average_difficulty last_taught=c.semester_last_taught content=c.description prefix='course' pk=c.pk %}
-                    </li>
-                {% endfor %}
+            <ul class="course-list" id="courses-sd-{{sd.id}}">
             </ul>
+            <div class="text-center" id="spinner-sd-{{sd.id}}">
+                <div class="spinner-border" role="status" >
```

Comment:
Hmm, wonder if it'll look better if the spinner was in a white card instead of being directly on the background of the page

Reply:
That's probably better. Ideally, it's best to have a placeholder, and then replace it with either the actual data or a box indicating the empty state. An example of placeholders from Instagram:
![image](https://user-images.githubusercontent.com/7676354/97330486-a3251700-18bb-11eb-9622-8b4ab91a4704.png)
Since the main purpose of this PR is not design improvement, we can do that in another PR.

Reply:
@j-alicia-long We can track this in https://github.com/thecourseforum/theCourseForum2/issues/191.